pkg/apis/servicecatalog/types.go

```diff
@@ -64,34 +64,34 @@ type ServiceBrokerSpec struct {
 
 	// RelistBehavior specifies the type of relist behavior the catalog should
 	// exhibit when relisting ServiceClasses available from a broker.
-	RelistBehavior ServiceBrokerRelistBehavior
+	RelistBehavior ClusterServiceBrokerRelistBehavior
 
 	// RelistDuration is the frequency by which a controller will relist the
-	// broker when the RelistBehavior is set to ServiceBrokerRelistBehaviorDuration.
+	// broker when the RelistBehavior is set to ClusterServiceBrokerRelistBehaviorDuration.
 	RelistDuration *metav1.Duration
 
 	// RelistRequests is a strictly increasing, non-negative integer counter that
 	// can be manually incremented by a user to manually trigger a relist.
 	RelistRequests int64
 }
 
-// ServiceBrokerRelistBehavior represents a type of broker relist behavior.
-type ServiceBrokerRelistBehavior string
+// ClusterServiceBrokerRelistBehavior represents a type of broker relist behavior.
+type ClusterServiceBrokerRelistBehavior string
```

Comment:
I don't think these constants (or the typedef) need to be renamed because they can just be re-used by the namespaced variant

Reply:
done

Reply:
reverted